metric-anomaly-detector/src/main/java/org/hypertrace/alert/engine/metric/anomaly/detector/evaluator/QueryRequestHandler.java

```diff
@@ -64,4 +65,16 @@ QueryRequest getQueryRequest(
     return metricQueryBuilder.buildMetricQueryRequest(
         metricAnomalyEventCondition.getMetricSelection(), startTime, endTime, tenantId);
   }
+
+  Iterator<ResultSetChunk> executeQuery(
+      Map<String, String> requestHeaders,
+      MetricSelection metricSelection,
+      String tenantId,
+      long startTime,
+      long endTime) {
+    return queryServiceClient.executeQuery(
+        metricQueryBuilder.buildMetricQueryRequest(metricSelection, startTime, endTime, tenantId),
+        requestHeaders,
```

Comment:
Do we now need the above `executeQuery` and `getQueryRequest` functions ?

Reply:
missed removing it